### PR TITLE
Revert "editing date field, predicate is now verbatimEventDate and label is 'EDTF Date'"

### PR DESCRIPTION
**Pull request by no-reply**
The predicate was correct before, we need a follow up to reinstate the new
label.

This reverts commit 8598f9ef71bace7275dc66f65c60cba212f7b208.

### CODE REVIEW

**Comment by no-reply** [comment]
@jenlindner Can you look at why this revert is failing? It looks like the commit it reverts might have changed things unrelated to the work.

File: spec/features/edit_etd_spec.rb [hunk 1]
```diff
@@ -35,6 +35,7 @@
       fill_in 'Description', with: etd[:description].first
       # term for license URI set in factory
       select('Creative Commons BY-SA Attribution-ShareAlike 4.0 International', from: 'License')
+      fill_in 'Publisher', with: etd[:publisher].first
```
Comment (no-reply) [inline spec/features/edit_etd_spec.rb:38]: @jenlindner is this related to the date work? Or do we need to remove it separately to push this forward?

Comment (jenlindner) [reply spec/features/edit_etd_spec.rb:38]: No, we need to remove it separately. Or instead of reverting this commit, I could add another that changes the predicate back in core_metadata.rb and its spec.
